systems/subscriber/sim-manager/cmd/server/main.go

```diff
@@ -122,7 +122,7 @@ func runGrpcServer(gormDB sql.Db) {
 	grpcServer.ExtraUnaryInterceptors = []grpc.UnaryServerInterceptor{
 		fsInterceptor.UnaryServerInterceptor}
 
-	go msgBusListener(mbClient)
+	// go msgBusListener(mbClient)
```

Comment:
Does this work ? Are you able to publish messages without registering ?

Reply:
I forgot to uncomment it after testing, which was my mistake. It has now been resolved.